assets/sass/okta-theme.scss

```diff
@@ -733,6 +738,10 @@
       &[type="password"] {
       /* -- Input Fields: password input field -- */
         font-family: Verdana, sans-serif;
+
+        &:placeholder-shown {
```

Comment:
Fixes [OKTA-176404](https://oktainc.atlassian.net/browse/OKTA-176404) . This is because we use a different font for password field https://github.com/okta/okta-signin-widget/blob/master/assets/sass/okta-theme.scss#L735. Setting the font to `proxima` when we field is empty to avoid the jump seen in the label.

Reply:
https://okta.box.com/s/138r94bkg0lb34asgasf97abzhs9pdmo

Reply:
This is specific to the widget. We are doing this because we use a different font for the password field in the widget https://github.com/okta/okta-signin-widget/blob/master/assets/sass/okta-theme.scss#L735.

Reply:
wait, so all our fonts in the widget are the same EXCEPT for the password field? Any chance we can then make all consistent?

Reply:
Unfortunately yes. We have had a long discussion about this already on slack (show_password_toggle). We cant make the password field's font the same as the other inputs as that font (proxima) makes the bullets look really tiny. 

<img width="435" alt="screen shot 2018-06-29 at 11 14 28 am" src="https://user-images.githubusercontent.com/23267876/42245769-deb44536-7ece-11e8-8cf8-c76eb6097681.png">